docs/architecture/adr-005-modular-server.md

```diff
@@ -0,0 +1,169 @@
+# ADR 005: Modular Server
```

Comment:
There was a gap - 005 didn't exist before?

Reply:
yeah it didn't exist